### PR TITLE
Rename LICENCE-2.0 to APACHE-2.0

### PR DESCRIPTION
**Pull request by subsidel**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -12,7 +12,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
```
Comment (tsadler1988) [inline LICENSE]: As this is a link to the Apache website, this should not have been changed.

Comment (subsidel) [inline LICENSE]: Good spot

File: LICENSE [hunk 2]
```diff
@@ -12,7 +12,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
...
+    http://www.apache.org/licenses/LICENCE-2.0
```
Comment (dhurrell) [inline LICENSE]: This is still wrong. It should be the US spelling, "license" (as it was originally).

Comment (subsidel) [inline LICENSE]: I am the worst at spelling, apparently :cry:

**Pull request merged**